<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>WeatherForecaster</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/new/prefix1/res/icon.ico</normaloff>:/new/prefix1/res/icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #F0F0F0;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>75</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>75</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Microsoft YaHei UI&quot;;
background-color: transparent;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>welcome</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cityComboBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox
{
padding: 5px;
font: 12pt &quot;Microsoft YaHei UI&quot;;
background-color: transparent;
color: rgb(255, 255, 255);
}

QComboBox::QAbstractItemView 
{
	background-color: rgb(255, 255, 255);
}
QComboBox:hover
{
    padding-left:3px;
    padding-top:3px;
	color: rgb(255, 0, 0);
}
</string>
   </property>
   <property name="placeholderText">
    <string>选择一个城市</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	image: url(:/new/prefix1/res/query.png);
background-color:transparent;
}
QPushButton:hover
{
    padding-left:3px;
    padding-top:3px;
}
QPushButton:pressed
{
    padding-left:3px;
    padding-top:3px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="resultTable">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>1000</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border:none;
background-color: transparent;
font: 700 10pt &quot;Microsoft YaHei UI&quot;;
color: rgb(255, 255, 255);


</string>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>100</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>100</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>30</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>30</number>
   </attribute>
   <column>
    <property name="text">
     <string>日期</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>星期</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>天气</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>最高温</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>最低温</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>日出时间</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>日落时间</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>空气质量</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>风向</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>风力</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>60</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>230</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>230</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Microsoft YaHei UI&quot;;
background-color: transparent;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>当前城市：尚未查询</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEdit">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>0</y>
     <width>130</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Meiryo UI&quot;;
border: none;
background-color: transparent;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="displayFormat">
    <string>H:mm:ss</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>0</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Meiryo UI&quot;;
background-color: transparent;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Time : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>0</y>
     <width>600</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Microsoft YaHei UI&quot;;
background-color: transparent;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>下方可查询未来15天的天气状况。Tips: 点击表头可排序</string>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/res/bg.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="historyButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>150</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
image: url(:/new/prefix1/res/history.png);
background-color:transparent;
}
QPushButton:hover
{
    padding-left:3px;
    padding-top:3px;
}
QPushButton:pressed
{
    padding-left:3px;
    padding-top:3px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>150</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	image: url(:/new/prefix1/res/exit.png);
background-color:transparent;
}
QPushButton:hover
{
    padding-left:3px;
    padding-top:3px;
}
QPushButton:pressed
{
    padding-left:3px;
    padding-top:3px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="chartButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>150</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
image: url(:/new/prefix1/res/chart-line.png);
background-color:transparent;
}
QPushButton:hover
{
    padding-left:3px;
    padding-top:3px;
}
QPushButton:pressed
{
    padding-left:3px;
    padding-top:3px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labeltable">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>800</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 48pt &quot;Microsoft YaHei UI&quot;;
color: rgb(255, 255, 255);
background-color: transparent;
border: 5px solid rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;当前还未查询&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>label</zorder>
  <zorder>cityComboBox</zorder>
  <zorder>searchButton</zorder>
  <zorder>resultTable</zorder>
  <zorder>label_2</zorder>
  <zorder>timeEdit</zorder>
  <zorder>label_4</zorder>
  <zorder>label_3</zorder>
  <zorder>historyButton</zorder>
  <zorder>closeButton</zorder>
  <zorder>chartButton</zorder>
  <zorder>labeltable</zorder>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
